src/main/java/screens/MazeDesignerUI.java

```diff
@@ -0,0 +1,176 @@
+package screens;
+
+import design.MazeDesignerController;
+import design.MazeDesignerInteractor;
+import design.MazeDesignerPresenter;
+import javafx.application.Application;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ToggleButton;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.stage.Popup;
+import javafx.stage.Stage;
+import javafx.geometry.Pos;
+
+public class MazeDesignerUI extends Application {
+    String css = this.getClass().getResource("/stylesheet.css").toExternalForm();
+    final MazeDesignerController mdc = new MazeDesignerController(new MazeDesignerInteractor(new MazeDesignerPresenter()));
+
```

Comment:
Add a constructor for this class that takes in a controller instance 

```suggestion
public MazeDesignerUI(MazeDesignerController mdc) {
    this.mdc = mdc;
}
```